example.jsonnet

```diff
@@ -1,12 +1,16 @@
 local k = import "ksonnet.beta.3/k.libsonnet";
+local service = k.core.v1.service;
+local servicePort = k.core.v1.service.mixin.spec.portsType;
 local grafana = import "grafana/grafana.libsonnet";
-local namespace = "monitoring";
+local namespace = "monitoring-grafana";
```

Comment:
Why did you change this namespace? I know you're supposed to copy this file, but asking anyway.